src/models/generated/GenericDER.jl

```diff
@@ -0,0 +1,403 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
```

Comment:
A minor comment is to include a references to paper that defines this model in the docstring